docs/multiplatform.md

```diff
@@ -4,82 +4,58 @@ This document covers using kotlin-inject in a multiplatform project. The example
 setting up an android and iOS project but the pattern should be the same for other platforms. You can check out
 [greeter](https://github.com/evant/kotlin-inject-samples/tree/main/multiplatform/greeter) for a full sample.
 
-## Download
+## Gradle
 
-Using [ksp](https://github.com/google/ksp)
+See the [README](https://github.com/evant/kotlin-inject/blob/main/README.md) for instructions on how to add kotlin-inject to your project.
 
-`settings.gradle`
-
-```kotlin
-pluginManagement {
-    repositories {
-        mavenCentral()
-        gradlePluginPortal()
-    }
-}
-
-dependencyResolutionManagement {
-    repositories {
-        mavenCentral()
-        google()
-    }
-}
```

Comment:
I personally use this in my projects but it's still an unstable API (marked as `@Incubating` in Gradle)